cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
> minishift hostfolder setup-users

Ahh, now I understand what it does. Somehow the name implied to me that you are setting up some user credentials. Could we make this somehow an option of 'minishift hostfolder add'? 'minishift hostfolder add --user-home'? I think I'd find this more intuitive.